### DISCUSSION TITLE
(pre-proposal)  Versioned Docs

### DISCUSSION THREAD

**Discussion by tomByrer** [withastro/roadmap#432] [Proposal]
Seems there aren't too many deprecations or new best-practices from v1 to v2, but I'll assume there will be v2 to v3, or maybe between v2.0 vs v2.6.   But many places will stay with older versions for one reason or another.

To help people who are stuck on older Astro versions, could you have your docs versioned please?

**Comment by florian-lefebvre** [comment]
We do not maintain versioned docs but we keep a deployment of older docs: https://v4.docs.astro.build/en/getting-started/
